web/src/lib/components/ui/constraintSignature/constraint-signature.svelte

```diff
@@ -172,19 +99,19 @@
       <p>{parameter.parameterName}</p>
       <div class="text-primary">
         {#if parameter.parameterType === ParameterType.Teacher}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={teacherItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {:else if parameter.parameterType === ParameterType.Room}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={roomItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {:else if parameter.parameterType === ParameterType.Grade}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={gradeItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {:else if parameter.parameterType === ParameterType.Tag}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={tagItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {:else if parameter.parameterType === ParameterType.Timeslot}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={timeslotItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {:else if parameter.parameterType === ParameterType.StudentGroup}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={stundeGroupItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {:else if parameter.parameterType === ParameterType.Subject}
-          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} data={subjectItems} />
+          <ComboBox onSearch={(value) => updateItems(value, parameter.parameterType)} {data} />
         {/if}
```

Comment:
This doesn't make sense.
1. The ifs are now useless
2. You are giving every ComboBox the exact same data, but when you load the page for the first time, no ComboBox will be open and no search request will be executed. This way all of the ComboBoxes would have the same content